### PR TITLE
Feature/cancel unbond

### PR DESCRIPTION
**Pull request by timlind**

### CODE REVIEW

File: x/clp/keeper/msg_server.go [hunk 1]
```diff
@@ -37,6 +37,45 @@ func (k msgServer) SetSymmetryThreshold(goCtx context.Context, threshold *types.
...
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	lp, err := k.Keeper.GetLiquidityProvider(ctx, request.ExternalAsset.Symbol, request.Signer)
+	if err != nil {
+		return nil, types.ErrLiquidityProviderDoesNotExist
+	}
+	// Prune unlocks
+	params := k.GetRewardsParams(ctx)
+	k.PruneUnlockRecords(ctx, &lp, params.LiquidityRemovalLockPeriod, params.LiquidityRemovalCancelPeriod)
+
+	err = k.UseUnlockedLiquidity(ctx, lp, request.Units)
```
Comment (timlind) [inline x/clp/keeper/msg_server.go:58]: This needs to use unbond-records in any state.

**Review by AustinoBombino — COMMENTED** [review]
logic looks good to me.

do we need a rest tx for this?

**Review by canercandan — APPROVED** [review]
LGTM

**Comment by timlind** [comment]
Looks like we skipped the rest endpoint for unbond as well so unless FE specifically needs it we should be good without.

**Pull request merged**